webcomponents/tool/src/main/frontend/.storybook/main.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
   "stories": [
     "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)",
```

Comment:
Duplicated?

Reply:
No, they're different suffixes, but they can be on one line, good spot!

Reply:
We can get rid of the jsx, ts and tsx suffixes as we don't actually use those formats for stories. It was auto generated.